tests/unit/dojson/test_dojson_hepnames.py

```diff
@@ -1146,6 +1205,47 @@ def test_private_notes_from_595__a_9():
     assert expected == result['595']
 
 
+def test_urls_from_8564_a():
+    schema = load_schema('authors')
+    subschema = schema['properties']['urls']
+
+    snippet = (
+        '<record>'
+        '  <datafield tag="856" ind1="4" ind2=" ">'
+        '    <subfield code="u">http://www.haydenplanetarium.org/tyson/</subfield>'
+        '  </datafield>'
+        '  <datafield tag="856" ind1="4" ind2=" ">'
+        '    <subfield code="g">active</subfield>'
```

Comment:
I guess the curator meant "this is currently his Twitter account", and reused the semantics of the `100__g` field for this purpose. Doesn't matter anyway, this is ignored by DoJSON.